jest.config.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
```

Comment:
it's automatically generated by `jest --init` but I guess we can remove the commented-out sections, I find it's helpful to know what's configurable at a glance rather than having to refer to the configuration reference though.

Reply:
I'm ambivalent, so it's up to you :)